src/Fixer/FunctionNotation/FunctionDeclarationFixer.php

```diff
@@ -24,11 +26,53 @@
  *
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
  */
-final class FunctionDeclarationFixer extends AbstractFixer
+final class FunctionDeclarationFixer extends AbstractFixer implements ConfigurableFixerInterface
 {
+    const SPACING_NONE = 'none';
```

Comment:
Rationale for using the constants is here: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2459#discussion_r95901213

Originally, I wanted to use private `const`s but apparently they are only supported by PHP 7.1: https://3v4l.org/ZKbBJ